scipy/stats/tests/test_sampling.py

```diff
@@ -52,10 +54,12 @@ def support(self):
 all_methods = [
     ("TransformedDensityRejection", {"dist": StandardNormal()}),
     ("DiscreteAliasUrn", {"dist": [0.02, 0.18, 0.8]}),
-    ("NumericalInversePolynomial", {"dist": StandardNormal()})
+    ("NumericalInversePolynomial", {"dist": StandardNormal()}),
+    ("NaiveRatioUniforms", {"dist": StandardNormal(),
+                            "u_min": -np.exp(-0.5) * np.sqrt(2),
+                            "u_max": np.exp(-0.5) * np.sqrt(2), "v_max": 1.0})
```

Comment:
I don't know the status of UNU.RAN testing, but `NumericalInverseHermite` tests with all continuous distributions (minus a few exceptions). It looks like `NaiveRatioUniforms` will only test with `StandardNormal` here; is that right?

Reply:
The reason is that you need to provide the bounds `u_min, ...`. if we make these parameters optional, one could run more test cases easily. But then it is mainly a test if the numerical optimization works :)